.github/workflows/deploy-guessing-game-temp.yml

```diff
@@ -0,0 +1,30 @@
+name: Deploy Number Guessing Game-Temporary Fix
```

Comment:
What's this fix ?

Reply:
Removed installing Hardhat, Compiling & Deploying as the previous workflow involved Hardhat@esm which has deprecated & fails the build.

Reply:
Let's change the name and stop using temp.

Reply:
If this is going to be the norm, lets change the name and remove fix and temp.

Artifacts *should* be built as part of the flow and *not* commit them in the repo. If there's a bug with hh it should be fixed.